integration-tests/subscriptions.test.ts

```diff
@@ -31,44 +27,81 @@ test('empty test placeholder', () => {
 });
 
 if (SUBSCRIPTIONS_ENABLED === 'true') {
+    if (!SUBSCRIPTIONS_ENDPOINT) {
+        throw new Error('SUBSCRIPTIONS_ENDPOINT environment variable is not defined');
+    }
+    let client: AxiosInstance;
+
+    const subscriptionResource = {
+        resourceType: 'Subscription',
+        status: 'requested',
+        // get a time 10 seconds (10000 ms) in the future
+        end: new Date(new Date().getTime() + 10000).toISOString(),
+        reason: 'Monitor Patients with name Smith',
+        criteria: 'Patient?name=Smith',
+        channel: {
+            type: 'rest-hook',
+            endpoint: SUBSCRIPTIONS_ENDPOINT!,
```

Comment:
This happens on L30-L32, but I can move it next to the API_AWS_REGION or inside the test itself to make it more clear.

Reply:
my bad. Still, it's probably fine to move it to the top of the file since other tests will use it. 
Also the `!` is not needed then on `SUBSCRIPTIONS_ENDPOINT!` if there is already an undefined check